src/domain/registration/permissions.ts

```diff
@@ -13,3 +14,24 @@ export const canUserUpdateSignupPresenceStatus = ({
     user?.is_strongly_identified && registration?.has_registration_user_access
   );
 };
+
+export const checkCanUserDoRegistrationAction = ({
+  action,
+  registration,
+  user,
+}: {
+  action: REGISTRATION_ACTIONS;
+  registration?: Registration;
+  user?: User;
+}): boolean => {
+  const isRegistrationUser = hasRegistrationUserAccess({
```

Comment:
Are superuser and registration admin permissions handled somewhere else or should they also be allowed to do the actions here?

Reply:
Superuser and registration admin users are using these same features from Linked Events UI e.g. https://linkedevents.dev.hel.ninja/fi/registrations/29/signups?returnPath=%2Fregistrations. Permissions for them are checked there. The reason to have separate page for registration user access user it that Linked Events UI doesn't support Suomi.fi authentication so users are not strongly identified.